iset.mm

```diff
@@ -98005,6 +98021,113 @@ This definition (in terms of ` U. ` and ` ~<_ ` ) is not taken directly
+    LUNRTUAURUOGSKURGGGUBUCZEFZUOGUDCVAGSUEGUFQURUTUNEGUGUHUIQUJ $.
+
+  $( A set with only one element is equinumerous to the ordinal number 1.
+     (Contributed by Jim Kingdon, 23-Feb-2022.) $)
+  sizeen1 $p |- ( A e. Fin -> ( ( size ` A ) = 1 <-> A ~~ 1o ) ) $=
+    ( cfn wcel csize cfv c1 c0 csn cen wbr c1o cvv 0ex sizesng ax-mp eqcomi a1i
+    wceq eqeq2d wb snfig sizeen mpan2 df1o2 breq2i 3bitrd ) ABCZADEZFRUHGHZDEZR
+    ZAUIIJZAKIJZUGFUJUHFUJRUGUJFGLCZUJFRMGLNOPQSUGUIBCZUKULTUNUOMGLUAOAUIUBUCUL
+    UMTUGUIKAIKUIUDPUEQUF $.
+
+  $( A function on a finite set is equinumerous to its domain.  (Contributed by
+     Jim Kingdon, 24-Feb-2022.) $)
+  sizefn $p |- ( ( F Fn A /\ A e. Fin ) -> ( size ` F ) = ( size ` A ) ) $=
+    ( wfn cfn wcel wa csize cfv wceq cen fndmeng ensymd wb fnfi sizeen sylancom
+    wbr mpbird ) BACZADEZFZBGHAGHIZBAJQZUAABADBKLSTBDEUBUCMABNBAOPR $.
+
+  $( The value of the size function on a finite 1-based sequence.  (Contributed
+     by Jim Kingdon, 24-Feb-2022.) $)
+  fseq1size $p |- ( ( N e. NN0 /\ F Fn ( 1 ... N ) ) -> ( size ` F ) = N ) $=
+    ( cn0 wcel c1 cfz co wfn csize cfv cfn wceq simpr simpl nn0zd fzfigd sizefn
+    wa 1zzd syl2anc sizefz1 adantr eqtrd ) BCDZAEBFGZHZRZAIJZUEIJZBUGUFUEKDUHUI
+    LUDUFMUGEBUGSUGBUDUFNOPUEAQTUDUIBLUFBUAUBUC $.
+
+  ${
+    $d A n z $.  $d B n $.  $d G n z $.
+    omgadd.1 $e |- G = frec ( ( x e. ZZ |-> ( x + 1 ) ) , 0 ) $.
+    $( Mapping ordinal addition to integer addition.  (Contributed by Jim
+       Kingdon, 24-Feb-2022.) $)
+    omgadd $p |- ( ( A e. _om /\ B e. _om ) ->
+                     ( G ` ( A +o B ) ) = ( ( G ` A ) + ( G ` B ) ) ) $=
+      ( com wcel coa co cfv caddc wceq wi c0 oveq2 fveq2d oveq2d cc0 cn0 c1 0zd
+      vn vz cv csuc fveq2 eqeq12d imbi2d wf1o wf frechashgf1o f1of ax-mp nn0cnd
+      ffvelrni addid1d frec2uz0d 3eqtr4rd w3a wa nnasuc nnacl frec2uzsucd eqtrd
+      nna0 3adant3 3ad2ant1 3ad2ant2 1cnd addassd oveq1 3ad2ant3 3eqtr4d 3expia
+      cc id expcom a2d finds impcom ) CFGBFGZBCHIZDJZBDJZCDJZKIZLZWABUBUDZHIZDJ
+      ZWDWHDJZKIZLZMWABNHIZDJZWDNDJZKIZLZMWABUCUDZHIZDJZWDWSDJZKIZLZMWABWSUEZHI
+      ZDJZWDXEDJZKIZLZMWAWGMUBUCCWHNLZWMWRWAXKWJWOWLWQXKWIWNDWHNBHOPXKWKWPWDKWH
+      NDUFQUGUHWHWSLZWMXDWAXLWJXAWLXCXLWIWTDWHWSBHOPXLWKXBWDKWHWSDUFQUGUHWHXELZ
+      WMXJWAXMWJXGWLXIXMWIXFDWHXEBHOPXMWKXHWDKWHXEDUFQUGUHWHCLZWMWGWAXNWJWCWLWF
+      XNWIWBDWHCBHOPXNWKWEWDKWHCDUFQUGUHWAWDRKIWDWQWOWAWDWAWDFSBDFSDUIFSDUJADEU
+      KFSDULUMZUOUNZUPWAWPRWDKWAARDWAUAEUQQWAWNBDBVEPURWSFGZWAXDXJWAXQXDXJMWAXQ
+      XDXJWAXQXDUSZXGXATKIZXIWAXQXGXSLXDWAXQUTZXGWTUEZDJXSXTXFYADBWSVAPXTAWTRDX
+      TUAEBWSVBVCVDVFXRXCTKIZWDXBTKIZKIZXSXIXRWDXBTWAXQWDVOGXDXPVGXQWAXBVOGXDXQ
+      XBFSWSDXOUOUNVHXRVIVJXDWAXSYBLXQXAXCTKVKVLXQWAXIYDLXDXQXHYCWDKXQAWSRDXQUA
+      EXQVPVCQVHVMVDVNVQVRVSVT $.
+  $}
+
+  ${
+    $d A m n x $.  $d B m n x $.
+    $( Dominance relation for the size function.  (Contributed by Jim Kingdon,
+       24-Feb-2022.) $)
```

Comment:
I would add "The size function preserves order, that is, it is a morphism from the finite sets with the dominance relation to the nonnegative integers with the less-than-or-equal-to relation".  But maybe I'm fussy, plus this comment does not capture the biconditional. So, your choice!

Reply:
As above I'll stick with the existing (terser) wording (which is taken from set.mm).